packages/fxa-settings/src/pages/ResetPassword/ResetPassword/index.tsx

```diff
@@ -15,6 +15,7 @@ import { InputText } from '../../../components/InputText';
 import CardHeader from '../../../components/CardHeader';
 import WarningMessage from '../../../components/WarningMessage';
 import LinkRememberPassword from '../../../components/LinkRememberPassword';
+import { MozServices } from '../../../lib/types';
```

Comment:
Unrelated, but while looking at the imports (as one does), I noticed this above on L5+L8. Not sure if you want to merge them. Feel free to laugh, ignore, and block me since this is wildly out of scope of the PR.

```js
/* L5 */ import { RouteComponentProps } from '@reach/router';
/* L8 */ import { useNavigate } from '@reach/router';
```

Reply:
-old man yells at cloud emoji while thinking about the prettier-sort package-